src/java/main/org/apache/zookeeper/ClientCnxn.java

```diff
@@ -409,23 +400,26 @@ public ClientCnxn(String chrootPath, HostProvider hostProvider, int sessionTimeo
 
         sendThread = new SendThread(clientCnxnSocket);
         eventThread = new EventThread();
+        initProperties();
+    }
 
+    private void initProperties() {
+        disableAutoWatchReset = zooKeeper.getClientConfig().getBoolean(
```

Comment:
We don´t need this variable any longer, we can simplify and just use `zooKeeper.getClientConfig().getBoolean(...)` instead.